src/PhpSpec/ObjectBehavior.php

```diff
@@ -32,6 +32,9 @@
  * @method void beConstructedThrough($factoryMethod, array $constructorArguments)
  * @method void beAnInstanceOf($class)
  * @method void shouldHaveType($type)
+ * @method void shouldImplement($class)
+ * @method void shouldBeAnInstanceOf($class)
+ * @method void shouldReturnAnInstanceOf($class)
```

Comment:
Oh ok, I just copied the method names from the "Type Matcher" section of the documentation: http://www.phpspec.net/en/latest/cookbook/matchers.html